docs/source/guides/adding-interactivity/components-with-state/_examples/isolated_state/main.py

```diff
@@ -29,14 +29,14 @@ def handle_more_click(event):
     url = sculpture["url"]
 
     return html.div(
-        html.button("Next", on_click=handle_next_click),
+        html.button({"onClick": handle_next_click}, "Next"),
         html.h2(name, " by ", artist),
         html.p(f"({bounded_index + 1} or {len(sculpture_data)})"),
-        html.img(src=url, alt=alt, style={"height": "200px"}),
+        html.img({"src": url, "alt": alt, "style": {"height": "200px"}}),
         html.div(
             html.button(
-                f"{('Show' if show_more else 'Hide')} details",
-                on_click=handle_more_click,
+                {"onClick": handle_more_click},
```

Comment:
Need to confirm throughout the docs we are using the new underscore syntax.

Probably need to update the script to handle this.

Reply:
Yup. I think we can do this in a separate PR. I guess we have one more before renaming.

Reply:
Granted using camelCase is completely valid.